core/server/worker/src/main/java/alluxio/worker/netty/BlockWriteRequestContext.java

```diff
@@ -29,7 +27,6 @@
 
   BlockWriteRequestContext(Protocol.WriteRequest request, long bytesReserved) {
     super(new BlockWriteRequest(request));
-    Preconditions.checkState(request.getOffset() == 0);
```

Comment:
Why removing this check?

Reply:
because previously we assume the context is always initialized when all args are valid. Now we always create a new context even if the args are invalid.

Reply:
Gotcha thanks.